lib/app/features/ion_connect/model/events_metadata.f.dart

```diff
@@ -69,10 +69,27 @@ class EventsMetadataData with _$EventsMetadataData implements EventSerializable
           if (pubkey != null) ImmutableEventReference(eventId: eventId, masterPubkey: pubkey),
     ].toList();
 
+    final payloadJson = Map<String, dynamic>.from(
+      jsonDecode(eventMessage.content) as Map<String, dynamic>,
+    )
+      // Ensure minimal required fields for parser resilience
+      ..putIfAbsent('id', () => List.filled(64, '0').join())
+      ..putIfAbsent('sig', () => List.filled(128, '0').join());
```

Comment:
`sig` is nullable in `EventMessage` and is it a real world case when encoded event message doesn't have `id`?